cmd/oras/push.go

```diff
@@ -85,14 +95,17 @@ Example - Push repository with manifest annotation file
 			return opts.ReadPassword()
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			opts.targetRef = args[0]
+			refs := strings.Split(args[0], ",")
+			opts.targetRef = refs[0]
+			opts.extraRefs = refs[1:]
 			opts.FileRefs = args[1:]
 			return runPush(opts)
 		},
 	}
 
 	cmd.Flags().StringVarP(&opts.manifestConfigRef, "config", "", "", "manifest config file")
 	cmd.Flags().StringVarP(&opts.artifactType, "artifact-type", "", "", "media type of the manifest config")
+	cmd.Flags().Int64VarP(&opts.concurrency, "concurrency", "", 5, "concurrency level")
```

Comment:
Concurrency should also be effective in copy options.

Reply:
done